src/components/gui/gui.jsx

```diff
@@ -36,6 +47,9 @@ const GUIComponent = props => {
         );
     }
 
+    const enableExtensions = window.location.search.includes('extensions');
+    const extensionButtonClasses = `${styles.extensionButton} ${enableExtensions ? '' : styles.hidden}`;
```

Comment:
I'm cool if you leave it this way for now, since it is obviously pretty temporary, but just an FYI we usually use the `classNames` helper for doing this. The syntax can seem funky at first, because it can take both class names and objects with {name: true/false} and usually uses the dynamic property definition thing from ES6. 

```javascript
import classNames from 'classnames';

<IconButton
    className={classNames(styles.extensionButton, {
        [styles.hidden]: window.location.search.includes('extensions')
    })}
/>
```

You can see a full example here, for instance https://github.com/LLK/scratch-gui/blob/develop/src/components/close-button/close-button.jsx#L8-L19